### ISSUE TITLE
Example for file transfer XEP-0096.

### ISSUE THREAD

**Issue by strannik-j** [fritzy/SleekXMPP#367]
Please write an example of sending and receiving a file using XEP-0096.

**Comment by anirudh-chhangani** [comment]
For the time being you can look at this issue : 
https://github.com/fritzy/SleekXMPP/issues/236

and this example : 
https://github.com/fritzy/SleekXMPP/tree/develop/examples/ibb_transfer

Those might give you a rough idea!!

**Comment by goodprogrammist** [comment]
Hello! Check this link: https://github.com/back2arie/Kalkun/blob/master/application/plugins/sms_to_xmpp/libraries/abhinavsingh-JAXL-5829c3b/xep/jaxl.0096.php

**Comment by thigg** [comment]
Yes that would be really usefull. The provided example is in php not in python.
